js/ui/data_grid/ui.data_grid.export.js

```diff
@@ -761,24 +779,49 @@ dataGridCore.registerModule('export', {
                 _getExportToolbarItems: function() {
                     const exportOptions = this.option('export');
                     const texts = this.option('export.texts');
+                    const formats = this.option('export.formats') ?? [];
+
+                    const excelEnabled = exportOptions.enabled && formats.includes('xlsx');
+                    const pdfEnabled = exportOptions.enabled && formats.includes('pdf');
+
                     const items = [];
 
-                    if(exportOptions.enabled) {
+                    if(excelEnabled) {
                         items.push({
-                            text: texts.exportAll,
+                            text: format(texts.exportAll, 'Excel'),
                             icon: DATAGRID_EXPORT_EXCEL_ICON,
                             onClick: () => {
                                 this._exportController.exportToExcel();
-                            }
+                            },
                         });
 
                         if(exportOptions.allowExportSelectedData) {
                             items.push({
-                                text: texts.exportSelectedRows,
-                                icon: DATAGRID_EXPORT_SELECTED_ICON,
+                                text: format(texts.exportSelectedRows, 'Excel'),
+                                icon: pdfEnabled ? DATAGRID_EXPORT_EXCEL_ICON : DATAGRID_EXPORT_SELECTED_ICON,
```

Comment:
Is it typo? Checking pdfEnabled in excel section? 
icon: pdfEnabled ? DATAGRID_EXPORT_EXCEL_ICON : DATAGRID_EXPORT_SELECTED_ICON,

Reply:
It was by design
Now changed because of different design of icon for "export selected to pdf"